src/main/flight/position.c

```diff
@@ -34,7 +34,7 @@
 #include "fc/runtime_config.h"
 
 #include "flight/position.h"
-#include "flight/position_control.h"
+#include "flight/autopilot.h"
```

Comment:
This can be excluded. Ideally it shouldn't be needed in position.c if we abstract correctly.
In other words: autopilot needs position to work but position should not need autopilot to work.
```suggestion
```

Reply:
removed, and compiles without errors - thanks